robots/cmd/kubevirtci-presubmit-creator/main.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2021 The KubeVirt Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
```

Comment:
Nit, not sure what is the convention, but could you please split the imports into blocks, the same as we do in kubevirt/kubevirt? One first block for std library import, a second block for third party imports and a third for local imports. As I see it helps to quickly identify dependencies, wdyt?